src/transports/HTTPTransport.ts

```diff
@@ -8,17 +8,20 @@ type CredentialsOption = "omit" | "same-origin" | "include"
 interface HTTPTransportOptions {
   credentials?: CredentialsOption
   headers?: Record<string, string>
+  fetch?: any
```

Comment:
I'm not really sure what to put here (not a typescript guy...) but the linter didn't like `Function`